--- pike7.5-7.5.3/lib/master.pike.in.orig	2003-03-18 12:23:33.000000000 +0100
+++ pike7.5-7.5.3/lib/master.pike.in	2003-03-20 16:53:13.000000000 +0100
@@ -864,6 +864,14 @@
 #if "¤include_prefix¤"[0]!='¤'
   include_prefix = "¤include_prefix¤";
 #endif
+
+  // Debian paths
+  add_include_path("/usr/local/include/pike");
+  add_include_path("/usr/local/include/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__);
+  add_module_path("/usr/local/lib/pike/site_pike");
+  add_module_path("/usr/local/lib/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__+"/site_pike");
+  add_module_path("/usr/local/share/pike/site_pike");
+  add_module_path("/usr/local/share/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__+"/site_pike");
 }
 
 
