diff -U3 -r -N pike7.4-7.4.35.orig/lib/master.pike.in pike7.4-7.4.35/lib/master.pike.in
--- pike7.4-7.4.35.orig/lib/master.pike.in	2003-12-03 23:07:20.000000000 +0100
+++ pike7.4-7.4.35/lib/master.pike.in	2003-12-04 00:16:47.000000000 +0100
@@ -817,6 +817,14 @@
 #if "¤include_prefix¤"[0]!='¤'
   include_prefix = "¤include_prefix¤";
 #endif
+
+  // Debian paths
+  add_include_path("/usr/local/include/pike");
+  add_include_path(sprintf("/usr/local/include/pike%0.1f", __REAL_VERSION__));
+  add_module_path("/usr/local/lib/pike/site_pike");
+  add_module_path(sprintf("/usr/local/lib/pike%0.1f/site_pike", __REAL_VERSION__));
+  add_module_path("/usr/local/share/pike/site_pike");
+  add_module_path(sprintf("/usr/local/share/pike%0.1f/site_pike", __REAL_VERSION__));
 }
 
 
