diff -U3 -Narp pike7.6-7.6.24/lib/master.pike.in pike7.6-7.6.24.new/lib/master.pike.in
--- pike7.6-7.6.24/lib/master.pike.in	2004-07-23 14:17:24 +0200
+++ pike7.6-7.6.24.new/lib/master.pike.in	2005-01-06 23:27:54 +0100
@@ -565,6 +565,14 @@ string normalize_path( string path )
 #else
   return replace(path,"\\","/");
 #endif
+
+  // Debian paths
+  add_include_path("/usr/local/include/pike");
+  add_include_path("/usr/local/include/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__);
+  add_module_path("/usr/local/lib/pike/site_pike");
+  add_module_path("/usr/local/lib/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__+"/site_pike");
+  add_module_path("/usr/local/share/pike/site_pike");
+  add_module_path("/usr/local/share/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__+"/site_pike");
 }
 
 //! Mapping containing the cache of currently compiled files.
