--- 7.7.orig/lib/master.pike.in	2006-06-19 10:45:06.000000000 +0200
+++ 7.7/lib/master.pike.in	2006-08-09 02:06:38.393725578 +0200
@@ -585,6 +585,14 @@ string normalize_path( string path )
 #else
   return replace(path,"\\","/");
 #endif
+
+  // Debian paths
+  add_include_path("/usr/local/include/pike");
+  add_include_path("/usr/local/include/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__);
+  add_module_path("/usr/local/lib/pike/site_pike");
+  add_module_path("/usr/local/lib/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__+"/site_pike");
+  add_module_path("/usr/local/share/pike/site_pike");
+  add_module_path("/usr/local/share/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__+"/site_pike");
 }
 
 //! Mapping containing the cache of currently compiled files.
