--- pike7.7-7.7.21.orig/lib/master.pike.in	2005-08-17 01:03:03.000000000 +0200
+++ pike7.7-7.7.21/lib/master.pike.in	2005-09-03 03:06:50.159517736 +0200
@@ -581,6 +581,14 @@ string normalize_path( string path )
 #else
   return replace(path,"\\","/");
 #endif
+
+  // Debian paths
+  add_include_path("/usr/local/include/pike");
+  add_include_path("/usr/local/include/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__);
+  add_module_path("/usr/local/lib/pike/site_pike");
+  add_module_path("/usr/local/lib/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__+"/site_pike");
+  add_module_path("/usr/local/share/pike/site_pike");
+  add_module_path("/usr/local/share/pike"+__REAL_MAJOR__+"."+__REAL_MINOR__+"/site_pike");
 }
 
 //! Mapping containing the cache of currently compiled files.
