--- pike7.4-7.4.315/lib/master.pike.in	2005-06-14 03:17:32.000000000 +0200
+++ pike7.4-7.4.315.new/lib/master.pike.in	2005-06-18 01:59:16.873370792 +0200
@@ -995,6 +995,14 @@ void create()
 #if "¤include_prefix¤"[0]!='¤'
   include_prefix = "¤include_prefix¤";
 #endif
+
+  // Debian paths
+  add_include_path("/usr/local/include/pike");
+  add_include_path(sprintf("/usr/local/include/pike%0.1f", __REAL_VERSION__));
+  add_module_path("/usr/local/lib/pike/site_pike");
+  add_module_path(sprintf("/usr/local/lib/pike%0.1f/site_pike", __REAL_VERSION__));
+  add_module_path("/usr/local/share/pike/site_pike");
+  add_module_path(sprintf("/usr/local/share/pike%0.1f/site_pike", __REAL_VERSION__));
 }
 
 
