diff -U3 -Narp pike7.7-7.7.21.orig/bin/smartlink pike7.7-7.7.21/bin/smartlink
--- pike7.7-7.7.21.orig/bin/smartlink	2002-07-09 03:13:13.000000000 +0200
+++ pike7.7-7.7.21/bin/smartlink	2005-09-03 03:05:02.497884792 +0200
@@ -147,9 +147,8 @@ if test x$# != x0 ; then
      LDOPTS="$LDOPTS -R$RPATH"
    ;;
    Linux\ 2.*)
-     if test x$LINKING != xno -a "x$RPATH" != x ; then
-      LDOPTS="$LDOPTS -Wl,-rpath,$RPATH"
-     fi
+     LD_LIBRARY_PATH="$RPATH"
+     export LD_LIBRARY_PATH
    ;;
    UnixWare\ 7.*)
      if test x$LINKING != xno -a "x$RPATH" != x ; then
diff -U3 -Narp pike7.7-7.7.21.orig/src/smartlink.c pike7.7-7.7.21/src/smartlink.c
--- pike7.7-7.7.21.orig/src/smartlink.c	2004-05-11 16:16:08.000000000 +0200
+++ pike7.7-7.7.21/src/smartlink.c	2005-09-03 03:05:02.498884640 +0200
@@ -150,7 +150,7 @@ int main(int argc, char **argv)
   full_rpath = rpath;
 
 #ifdef USE_Wl
-  strcat(rpath, "-Wl,-rpath,");
+  /* strcat(rpath, "-Wl,-rpath,"); */
 #elif defined(USE_Wl_R)
   strcat(rpath, "-Wl,-R");
 #elif defined(USE_R)
@@ -262,7 +262,7 @@ int main(int argc, char **argv)
     }
     rpath_in_use |= add_path(rpath, ld_lib_path);
   }
-
+#if 0
   if (rpath_in_use) {
     /* Delete the terminating ':' */
     rpath[strlen(rpath) - 1] = 0;
@@ -309,6 +309,7 @@ int main(int argc, char **argv)
 #error Unknown method
 #endif
   }
+#endif
 
   new_argv[new_argc++] = NULL;
 
