diff -U3 -r -N pike7.4-7.4.35.orig/bin/smartlink pike7.4-7.4.35/bin/smartlink
--- pike7.4-7.4.35.orig/bin/smartlink	2002-07-09 03:13:13.000000000 +0200
+++ pike7.4-7.4.35/bin/smartlink	2003-12-04 00:17:53.000000000 +0100
@@ -147,9 +147,8 @@
      LDOPTS="$LDOPTS -R$RPATH"
    ;;
    Linux\ 2.*)
-     if test x$LINKING != xno -a "x$RPATH" != x ; then
-      LDOPTS="$LDOPTS -Wl,-rpath,$RPATH"
-     fi
+     LD_LIBRARY_PATH="$RPATH"
+     export LD_LIBRARY_PATH
    ;;
    UnixWare\ 7.*)
      if test x$LINKING != xno -a "x$RPATH" != x ; then
diff -U3 -r -N pike7.4-7.4.35.orig/src/smartlink.c pike7.4-7.4.35/src/smartlink.c
--- pike7.4-7.4.35.orig/src/smartlink.c	2002-10-11 16:23:19.000000000 +0200
+++ pike7.4-7.4.35/src/smartlink.c	2003-12-04 00:17:53.000000000 +0100
@@ -146,7 +146,9 @@
   full_rpath = rpath;
 
 #ifdef USE_Wl
+#if 0
   strcat(rpath, "-Wl,-rpath,");
+#endif
 #elif defined(USE_Wl_R)
   strcat(rpath, "-Wl,-R");
 #elif defined(USE_R)
@@ -257,6 +259,7 @@
     rpath_in_use |= add_path(rpath, ld_lib_path);
   }
 
+  rpath_in_use = 0;
   if (rpath_in_use) {
     /* Delete the terminating ':' */
     rpath[strlen(rpath) - 1] = 0;
@@ -283,9 +286,11 @@
       new_argv[new_argc++] = rpath;
     }
 #elif defined(USE_Wl)
+#if 0
     if (linking) {
       new_argv[new_argc++] = full_rpath;
     }
+#endif
 #elif defined(USE_R) || defined(USE_Wl_R)
     new_argv[new_argc++] = full_rpath;
 #elif defined(USE_LD_LIBRARY_PATH)
@@ -335,7 +340,7 @@
       }
     }
   }
-
+ 
   execv(argv[1], new_argv);
   fprintf(stderr, "%s: exec of %s failed!\n", argv[0], argv[1]);
   exit(1);
