diff -U3 -r 7.3/src/modules/Gmp/Makefile.in 7.3.new/src/modules/Gmp/Makefile.in
--- 7.3/src/modules/Gmp/Makefile.in	2002-04-30 00:54:24.000000000 +0200
+++ 7.3.new/src/modules/Gmp/Makefile.in	2002-09-18 21:22:30.000000000 +0200
@@ -20,6 +20,6 @@
 next_prime.o: $(SRCDIR)/prime_table.out
 
 $(SRCDIR)/prime_table.out: $(SRCDIR)/prime_table.pike
-	$(BUILD_BASE)/precompile.sh $(SRCDIR)/prime_table.pike $(PRIME_LIMIT) >$(SRCDIR)/prime_table.out || { rm $(SRCDIR)/prime_table.out; exit 1; }
+	$(BUILD_BASE)/precompile.sh $(SRCDIR)/prime_table.pike $(PRIME_LIMIT) >$(SRCDIR)/prime_table.out || true
 
 @dependencies@
