--- 7.7.orig/src/Makefile.in	2006-08-09 01:30:56.000000000 +0200
+++ 7.7/src/Makefile.in	2006-08-09 02:12:35.898190075 +0200
@@ -56,7 +56,7 @@ man_prefix = $(prefix)/man
 # TESTARGS = -F -a
 
 # Arguments for install.pike used by the install targets.
-INSTALLARGS = --new-style
+INSTALLARGS ?= --new-style
 
 #
 # use bison please, yacc isn't good enough.
@@ -600,7 +600,7 @@ install: pike-complete-stamp tools
 	MANDIR_SRC="$(MANDIR_SRC)" \
 	man_prefix="$(man_prefix)" pike_name=$(pike_name) \
 	fakeroot="$(buildroot)" PIKE_MODULE_RELOC="$(PIKE_MODULE_RELOC)" \
-	$(INSTALLARGS)
+	include_prefix="$(include_prefix)" $(INSTALLARGS)
 
 install_interactive: pike-complete-stamp tools
 	@$(RUNPIKE) $(TMP_BINDIR)/install.pike --interactive \
