--- pike7.7-7.7.21.orig/src/Makefile.in	2005-08-17 01:03:07.000000000 +0200
+++ pike7.7-7.7.21/src/Makefile.in	2005-09-03 03:02:50.420963536 +0200
@@ -56,7 +56,7 @@ man_prefix = $(prefix)/man
 # TESTARGS = -F -a
 
 # Arguments for install.pike used by the install targets.
-INSTALLARGS = --new-style
+INSTALLARGS ?= --new-style
 
 #
 # use bison please, yacc isn't good enough.
@@ -534,7 +534,7 @@ install: pike tools
 	MANDIR_SRC="$(MANDIR_SRC)" \
 	man_prefix="$(man_prefix)" pike_name=$(pike_name) \
 	fakeroot="$(buildroot)" PIKE_MODULE_RELOC="$(PIKE_MODULE_RELOC)" \
-	$(INSTALLARGS)
+	include_prefix="$(include_prefix)" $(INSTALLARGS)
 
 install_interactive: pike tools
 	@$(RUNPIKE) $(TMP_BINDIR)/install.pike --interactive \
