--- 7.6/bin/install.pike	2005-12-30 14:33:48.450200000 +0100
+++ 7.6.fix/bin/install.pike	2005-12-30 15:14:24.806462750 +0100
@@ -1197,7 +1197,7 @@ int pre_install(array(string) argv)
       exec_prefix=combine_path(prefix,"bin");
       lib_prefix=combine_path(prefix,"lib");
       doc_prefix=combine_path(prefix,"doc");
-      include_prefix=combine_path(prefix,"include","pike");
+      include_prefix=vars->include_prefix||combine_path(prefix,"include","pike");
       man_prefix=combine_path(prefix,"man");
       if (export) {
 	low_install_file(combine_path(vars->TMP_BINDIR,"install.pike"),
