--- pike7.6-7.6.51/src/modules/dynamic_module_makefile.in	2004-04-23 17:41:34.000000000 +0200
+++ pike7.6-7.6.51.new/src/modules/dynamic_module_makefile.in	2005-11-18 21:26:17.094580000 +0100
@@ -28,7 +28,7 @@ modlist_segment: Makefile
 
 # Can't depend on $(SRCDIR)/$(CONFIG_HEADERS).in since
 # $(CONFIG_HEADERS) isn't always used.
-Makefile: $(MODULE_BASE)/dynamic_module_makefile $(SRCDIR)/Makefile.in $(SRCDIR)/dependencies make_variables config.status
+Makefile: $(PIKE_SRC_DIR)/dynamic_module_makefile $(SRCDIR)/Makefile.in $(SRCDIR)/dependencies make_variables config.status
 	CONFIG_FILES=Makefile CONFIG_HEADERS="$(CONFIG_HEADERS)" ./config.status
 	touch remake
 	@echo "Run make again" >&2
@@ -57,7 +57,7 @@ module.so: $(MODULE_ARCHIVES) $(OBJS)
 	  exit 1; \
 	fi
 
-$(OBJS) : $(MODULE_BASE)/dynamic_module_makefile
+$(OBJS) : $(PIKE_SRC_DIR)/dynamic_module_makefile
 
 #
 # install a standard module with optional c component in the system module path
