diff -U3 -Narp pike7.6-7.6.69/lib/modules/SSL.pmod/sslfile.pike pike7.6-7.6.69.ssl/lib/modules/SSL.pmod/sslfile.pike
--- pike7.6-7.6.69/lib/modules/SSL.pmod/sslfile.pike	2006-03-29 15:26:24.000000000 +0200
+++ pike7.6-7.6.69.ssl/lib/modules/SSL.pmod/sslfile.pike	2006-03-30 12:16:56.518253219 +0200
@@ -1479,6 +1479,8 @@ static int ssl_read_callback (int called
       SSL3_DEBUG_MSG ("ssl_read_callback: Calling accept callback %O\n",
 		      accept_callback);
       RESTORE;
+      if ( sizeof(read_buffer) )
+         real_backend->call_out (ssl_read_callback, 0, 1, 0);
       return accept_callback (this, callback_id);
     }
 
